lib/enkf/gen_obs.cpp

```diff
@@ -500,6 +500,20 @@ double gen_obs_iget_value(const gen_obs_type * gen_obs, int index){
   return gen_obs->obs_data[index];
 }
 
+void gen_obs_get_values(const gen_obs_type * gen_obs, int size, double * data) {
+      for (int i=0; i < size; i++) {
+        data[i] = gen_obs->obs_data[i];
+    }
+}
+
+
+void gen_obs_get_std(const gen_obs_type * gen_obs, int size, double * data) {
```

Comment:
get-function should return something, rename to "load" or something sensible

Reply:
:+1: